0.0.10/example.md

```diff
@@ -44,6 +46,9 @@ Explanation of parameters:
     - namespace: namespace in which the MemoryMachine is running, should be the same as the application namespace
 - spec: contains the configurations for MemoryMachine running in this namespace, details about available configurations can be found [in this document](config.md).
 
+**Note** 
+- Please be sure to have a default storageClass that supports dynamic provisioning. You can change the default storageClass in ```etcdConfig```. Please see [MemoryMachine Configuration List](config.md)
```

Comment:
nit: it is required to have a default one. When there is no default, user must specify one in `etcdConfig.storageClass`.
```suggestion
- Please be sure to have a StorageClass that supports dynamic provisioning. You can change the StorageClass in ```etcdConfig```. Please see [MemoryMachine Configuration List](config.md)
```